ddtrace/contrib/sqlite3/patch.py

```diff
@@ -26,7 +26,7 @@ def traced_connect(func, _, args, kwargs):
 
 def patch_conn(conn):
     wrapped = TracedSQLite(conn)
-    Pin(service="sqlite", app="sqlite").onto(wrapped)
+    Pin(service="sqlite", app="sqlite", app_type="db").onto(wrapped)
```

Comment:
can you update / add our tests with that? in that way we're sure to avoid regressions. Thank you!